.eslintrc

```diff
@@ -3,7 +3,6 @@ extends:
   - idiomatic
 env:
   browser: true
-  node: true
```

Comment:
When this is deleted it generates errors with `'module' is not defined` in each `config.js` file